data/json/items/armor/suits_protection.json

```diff
@@ -224,6 +224,750 @@
     "name": { "str": "XL plate armor" },
     "copy-from": "armor_lightplate",
     "proportional": { "weight": 1.125, "volume": 1.13, "price": 1.25 },
+    "flags": [ "VARSIZE", "OUTER", "STURDY", "OVERSIZE" ]
+  },
+  {
+    "id": "armor_lc_lightplate",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "mild steel 2 mm plate armor" },
+    "description": "A light suit of plate armor with a 2 mm thick chestpiece.  This one has been made with mild steel.",
+    "weight": "15 kg",
+    "volume": "17500 ml",
+    "price": 60000,
+    "price_postapoc": 12000,
+    "to_hit": -5,
+    "bashing": 8,
+    "material": [ { "type": "low_steel", "portion": 9 }, { "type": "leather", "portion": 1 } ],
```

Comment:
This doesn't do anything since you explicitly state thickness and coverage in the armor data should be:
`"material": [ "low_steel", "leather"]`

Reply:
If I remove it, it's showing the armor as zero defense since you can't see the locational data. Unless I've got that set up incorrectly, I haven't used it before now.